### ISSUE TITLE
Remove By situation button and add calculators

### ISSUE THREAD

**Issue by ButtercupUltra** [medic-code/IOL-Assist#40]
In order to avoid a touchy MEdical Device issue, we could ditch the By Situation button.

Instead, I'm tinking of a couple of tools that are also andy:
-IOLs should also have in their page the type of cartridge needed for their insertion, as well as the inyector
-Sulcus change carrousel
-Info regarding viscoelastics
-Tool to help determine size of capsular tension ring

**Comment by medic-code** [comment]
The By Situation might be one to put on the back burner as we're directly providing clinical guidance when thinking about this. Would need to think about how to do this, so things do not come back on us. Disclaimers etc... make sure we wont get into legal trouble if people use our app and something clinical concerning occurs.

With regard to the other tools worth discussing this in person! Can draw up some issues together to add to the list. Think we've discussed sulcus change carousel (This may be what I referred to it as https://github.com/medic-code/IOL-Assist/issues/24, can change the title for better clarity if I've got it right!)
